services/dqa-secondary-reports/main.py

```diff
@@ -0,0 +1,271 @@
+import codecs
+from datetime import datetime
+import git
+import json
+import logging
+import os
+import shutil
+import sys
+import uuid
+
+from requests.exceptions import HTTPError
+from flask import Flask, Response
+from flask.ext.cors import CORS
+
+from dictionary_parser import Parser as DictionaryParser
+from report_parser import Parser, logger
+from timer import ResourceTimer
+
+# Token for GitHub authorization. Set on startup.
+GITHUB_AUTH_TOKEN = None
+
+DIR_NAME = 'dqa_repo'
+REMOTE_URL = 'github.com/PEDSnet/Data-Quality.git'
+
+
+class DQAResource():
+    def __init__(self):
```

Comment:
It would be more flexible for this to take the `dir_name` rather than using a global.